tests/dicts/io/test_io_dict_ini.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+
+from benedict.dicts.io import IODict
+
+from .test_io_dict import io_dict_test_case
+
+TARGET_DICT = {
```

Comment:
The test dictionary should cover most use cases as possible:
-  Could you add at least one entry for each data-type?   
-  Even doing tests against both nested dicts and lists is recommended.
-  String entries containing special characters (eg. àèéìòùç) must be tested.

Reply:
The problem with .ini files is that they can only support a two level nested object, so the real world use case is also really limited.

Reply:
As for the datatypes, I have put str, float, int and boolean which are the types supported by ini files. Any other object will be converted to string.

Reply:
Ok, so there should be a validation that will raise an exception in case the dict that should be converted has more than 2 levels.